foundry.toml

```diff
@@ -4,7 +4,7 @@ names = true
 sizes = true
 via_ir = true
 optimizer = true
-optimizer_runs = 999999
+optimizer_runs = 200
```

Comment:
had to reduce this otherwise morpho contract deployment would throw `CreateContractSizeLimit`